content/docs/2.4/scalers/rabbitmq-queue.md

```diff
@@ -188,3 +192,45 @@ spec:
     authenticationRef:
       name: keda-trigger-auth-rabbitmq-conn
 ```
+
+#### HTTP protocol (`QueueLength`) and using regex (`useRegex`):
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: keda-rabbitmq-secret
+data:
+  host: <HTTP API endpoint> # base64 encoded value of format http://guest:password@localhost:15672/vhost
+---
+apiVersion: keda.sh/v1alpha1
+kind: TriggerAuthentication
+metadata:
+  name: keda-trigger-auth-rabbitmq-conn
+  namespace: default
+spec:
+  secretTargetRef:
+    - parameter: host
+      name: keda-rabbitmq-secret
+      key: host
+---
+apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
+metadata:
+  name: rabbitmq-scaledobject
+  namespace: default
+spec:
+  scaleTargetRef:
+    name: rabbitmq-deployment
+  triggers:
+  - type: rabbitmq
+    metadata:
+      protocol: http
+      queueName: testqueue
```

Comment:
Might be better to use some more complex regex in here as the example queueName, so it is more clear, where users should write the regex

Reply:
You are right, I have set `^.*incoming$` to clarify that it's a regex